test/quantization/test_quantize_fx.py

```diff
@@ -462,3 +465,26 @@ def test_layer_norm(self):
         data = (torch.rand((1, 2, 5, 5), dtype=torch.float),)
         self._test_norm_impl(
             nn.LayerNorm, F.layer_norm, [[2, 5, 5]], data, nnq.LayerNorm, torch.ops.quantized.layer_norm)
+
+    def test_instance_norm(self):
+        data_1d = (torch.rand((1, 4, 5), dtype=torch.float),)
+        data_2d = (torch.rand((1, 4, 5, 1), dtype=torch.float),)
+        data_3d = (torch.rand((1, 4, 5, 1, 1), dtype=torch.float),)
+        data_dict = {1 : data_1d, 2 : data_2d, 3 : data_3d}
+        instance_norm_modules = {1 : nn.InstanceNorm1d,
+                                 2 : nn.InstanceNorm2d,
+                                 3 : nn.InstanceNorm3d}
+        quantized_instance_norm_modules = {
+            1 : nnq.InstanceNorm1d,
+            2 : nnq.InstanceNorm2d,
+            3 : nnq.InstanceNorm3d
+        }
+        instance_norm_ops = {1 : F.instance_norm,
+                             2 : F.instance_norm,
+                             3 : F.instance_norm}
```

Comment:
nit: `Dict[int, object]` with `int` being sequential is effectively the same as a `List[object]`